urlchecker.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# url testing
+# source urlchecker.sh
+main_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+excluded_directory="*/node_modules/*"
+all_urls=()
+all_files=()
+match_file()
+{
```

Comment:
Grep supports printing files with matches:

```
└─ $ grep -rl "https://justice.gov" .
./crt_portal/cts_forms/locale/vi/LC_MESSAGES/django.mo
./crt_portal/cts_forms/locale/vi/LC_MESSAGES/django.po
./crt_portal/cts_forms/locale/zh_Hans/LC_MESSAGES/django.mo
./crt_portal/cts_forms/locale/zh_Hans/LC_MESSAGES/django.po
./crt_portal/cts_forms/locale/tl/LC_MESSAGES/django.mo
./crt_portal/cts_forms/locale/tl/LC_MESSAGES/django.po
./crt_portal/cts_forms/locale/all/LC_MESSAGES/django.po
./crt_portal/cts_forms/locale/zh_Hant/LC_MESSAGES/django.mo
./crt_portal/cts_forms/locale/zh_Hant/LC_MESSAGES/django.po
./crt_portal/cts_forms/locale/ko/LC_MESSAGES/django.mo
./crt_portal/cts_forms/locale/ko/LC_MESSAGES/django.po
./crt_portal/cts_forms/locale/es/LC_MESSAGES/django.mo
./crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po
./crt_portal/cts_forms/templates/vot_resources.html
```

Reply:
bump - you might be able to replace `print_files_for_url` with `grep -rl`, here.